UKHomeOffice/nmsw-ui
# NMSW-455 Update text on change your password page

# Ticket

NMSW-455

----

## AC

Change the support text on the Change your password page to match the supporting text on the Create your passoword page

----

## To test

- Run app
- Go to localhost:3000
- Sign in
- Click on the your details tab
- Click on change your password
- The support text should match the text on /create-account/your-password

----

## Notes
